pyproject.toml

```diff
@@ -0,0 +1,17 @@
+[tool.poetry]
+name = "mycompany-consent-demo"
+version = "0.1.0"
+description = ""
```

Comment:
Could we add a description, authors and license here (can be copied mostly from the file in the backend folder, but leave out the `backend` part of the description)?